src/slurm_plugin/clustermgtd.py

```diff
@@ -379,7 +380,7 @@ def _manage_compute_fleet_status_transitions(self):
                 self._compute_fleet_status,
             )
 
-    def manage_cluster(self):
+    def manage_cluster(self, cloudwatch_metrics):
```

Comment:
cloudwatch_metrics --> cluster_metrics.
The name of the parameter should not depend by the fact we'll store this info in cloudwatch. We can store them somewhere else.

Reply:
Done but changed to metrics_collector now.